spec/library_spec.rb

```diff
@@ -0,0 +1,30 @@
+require './lib/library.rb'
+
+describe Library do
+
+  let(:person) { double('Person')}
+
+  before do
+    allow(person).to receive(:perform_borrow)
+    .and_return({item: {title: "Alfons och soldatpappan", author: "Gunilla Bergström"}, available: false, return_date: Date.today + 30 })
+  end
+
+  it 'has a list of books on initialize' do
+    expect(subject.see_list).not_to be []
+  end
```

Comment:
Right. So I suspect that `subject.see_list` is `nil`, which is not `[]`, so this test goes green. Re-write this test so you can tell if it is actually doing what you think it's doing.